lib/util/git/index.ts

```diff
@@ -612,13 +612,15 @@ export type CommitFilesConfig = {
   files: File[];
   message: string;
   force?: boolean;
+  noVerify?: GitNoVerifyOption[];
 };
 
 export async function commitFiles({
   branchName,
   files,
   message,
   force = false,
+  noVerify = [],
```

Comment:
@rarkins I'm not sure if we should add this default value here 🤔

Reply:
I was uncertain whether I should default this to `['commit', 'push']` (same as config) or yeah, what to do. I'm neutral.

Reply:
```suggestion
  noVerify,
```

Reply:
Should I make the parameter required as well then?

Reply:
nvm I see what you mean